arkiverer/src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/fileservice/FileFetchUtilities.kt

```diff
@@ -16,8 +18,26 @@ fun mergeFetchResponsesAndSetOverallStatus(key: String, responses: List<FetchFil
 			exception = RuntimeException("$key: files not found ${responses.filter{it.status==ResponseStatus.NotFound.value }.map{it.files?.first()}.toList()}"))
 	else if (responses.any { it.status == ResponseStatus.Deleted.value })
 		FetchFileResponse(status = ResponseStatus.Deleted.value, files = responses.flatMap { it.files?:listOf() }.toList(),
-			exception = RuntimeException("$key: files not found ${responses.filter{it.status==ResponseStatus.Deleted.value }.map{it.files?.first()?.id}.toList()}"))
+			exception = RuntimeException("$key: files not found ${responses.filter{it.status==ResponseStatus.Deleted.value }.map{it.files?.first()?.uuid}.toList()}"))
 	else
 		FetchFileResponse(status = ResponseStatus.Error.value, files = responses.flatMap { it.files?:listOf() }.toList(),
 			exception = RuntimeException("$key: unexpected status mix ${responses.map{it.status + ',' + it.files?.first()?.status}.toList()}"))
 }
+
+fun mapToResponseStatus(fileStatus:SoknadFile.FileStatus ): ResponseStatus {
+	return when(fileStatus) {
+		SoknadFile.FileStatus.ok -> ResponseStatus.Ok
+		SoknadFile.FileStatus.notMinusFound -> ResponseStatus.NotFound
+		SoknadFile.FileStatus.deleted -> ResponseStatus.Deleted
+	}
+}
+
+fun mapToResponseStatus(fileStatus: String ): ResponseStatus {
```

Comment:
Kommenter forskjellen på disse på? Altså en for innsending-api og andre fra soknadsfillager